### DISCUSSION TITLE
How to auto populate fields based on other field value?

### DISCUSSION THREAD

**Discussion by keczanm** [formio/formio.js#4671] [Q&A]
Hi! 

First of all, thank you for your work! I'm checking the available demo at https://formio.github.io/formio.js/app/builder .

I was wondering if it is possible to autopopulate an input field based on another? 

For example it is often handy when the shipping and billing addresses are the same while placing an order. I would need a checkbox, like:

`[ ] Check if your shipping address is different from your billing address.`

Thanks for your help,
Matthew

**Comment by RajamArunachalam** [comment]
Here is an example, which takes two inputs such as Shipping Address and Billing Address and a check box to decide whether Shipping and Billing addresses are the same.

`{
    "display": "form",
    "settings": {
        "pdf": {
            "id": "1ec0f8ee-6685-5d98-a847-26f67b67d6f0",
            "src": "https://files.form.io/pdf/5692b91fd1028f01000407e3/file/1ec0f8ee-6685-5d98-a847-26f67b67d6f0"
        }
    },
    "components": [
        {
            "label": "Shipping Address",
            "tableView": true,
            "key": "shippingAddress",
            "type": "textfield",
            "input": true
        },
        {
            "label": "Same as Shipping Address",
            "tableView": false,
            "defaultValue": false,
            "key": "sameAddress",
            "type": "checkbox",
            "input": true
        },
        {
            "label": "Billing Address",
            "tableView": true,
            "calculateValue": "value = data.sameAddress ? data.shippingAddress : value;",
            "key": "billingAddress",
            "type": "textfield",
            "input": true
        },
        {
            "type": "button",
            "label": "Submit",
            "key": "submit",
            "disableOnInvalid": true,
            "input": true,
            "tableView": false
        }
    ]
}`

**Comment by keczanm** [comment]
Hi!

Thank you very much for your example, this helps a lot! :)

Best wishes,
Matthew
